contracts/SimpleManager.sol

```diff
@@ -136,14 +145,22 @@ contract SimpleManager is OwnableUpgradeable {
             "NM"
         );
         VaultInfo memory vaultInfo = vaults[vault_];
+        require(
+            vaultInfo.lastRebalanceTimestamp + vaultInfo.coolDownPeriod <
+                block.timestamp, // solhint-disable-line not-rely-on-time
+            "CP"
+        );
+
+        {
+            VaultInfo storage vaultInfoS = vaults[vault_];
+            // solhint-disable-next-line not-rely-on-time
+            vaultInfoS.lastRebalanceTimestamp = block.timestamp;
```

Comment:
Why not change to `vaults[vault_].lastRebalanceTimestamp = block.timestamp;` for simplicity?

Reply:
Sure, will change that.